PocketKit/Sources/Sync/Requests.swift

```diff
@@ -58,6 +58,19 @@ public enum Requests {
         return request
     }
 
+    public static func fetchSavedItems(bySearchTerm searchTerm: String, userPremium isPremium: Bool) -> NSFetchRequest<SavedItem> {
+        let request = SavedItem.fetchRequest()
+        let urlPredicate = NSPredicate(format: "url CONTAINS %@", searchTerm)
+        let titlePredicate = NSPredicate(format: "item.title CONTAINS %@", searchTerm)
+        let tagsPredicate = NSPredicate(format: "%@ IN tags.name", searchTerm)
```

Comment:
nitpick - maybe add `tagsPredicate` in the conditional check, so we don't create this object unless necessary

Reply:
ok